src/utils/firebaseServices.js

```diff
@@ -180,3 +184,26 @@ export const passwordResetHandler = (email, setAlert) => {
 		}));
 	}
 };
+
+export const getAllQuizQuestions = async (
+	setAlert,
+	allQuizQuestions,
+	setAllQuizQuestions
+) => {
+	const dbRef = realTimeDBRef(firebaseRealtimeDB);
+	try {
+		const allQuestions = await get(child(dbRef, "quizDb"));
+		setAllQuizQuestions(allQuestions.val());
+
+		if (allQuizQuestions.length > 1) {
+			localStorage.setItem(ROUTES.quizQuestionData, allQuestions.val());
```

Comment:
this is good that you made this kind of stuff using objects

Reply:
Thanks